lib/screens/wallet/components/receive_send_buttons.dart

```diff
@@ -20,66 +20,56 @@ class ReceiveSendButtons extends StatelessWidget {
             mainAxisAlignment: MainAxisAlignment.center,
             children: [
               Expanded(
-                child: Opacity(
-                  opacity: state.selectedIndex == 0 ? 1 : 0.3,
-                  child: MaterialButton(
-                    padding: const EdgeInsets.only(top: 14, bottom: 14),
-                    onPressed: state.selectedIndex == 0
-                        ? () => NavigationService.of(context).navigateTo(Routes.transfer)
-                        : null,
-                    color: AppColors.green1,
-                    disabledColor: AppColors.green1,
-                    shape: const RoundedRectangleBorder(
-                      borderRadius: BorderRadius.only(
-                        topLeft: Radius.circular(4),
-                        topRight: Radius.circular(50),
-                        bottomLeft: Radius.circular(50),
-                        bottomRight: Radius.circular(4),
-                      ),
+                child: MaterialButton(
```

Comment:
This changes are setting the buttons back to normal before we had to disable them?

Reply:
Yes that seems they are enabled now, I mean we have these new token ready to use right?

Reply:
Yes you can now send the tokens ;)

Reply:
Changes:
- removed Opacity widget
- changed onPressed to go to transfer for all tokens (was null for non-seeds tokens before)